gems/actionview/CVE-2023-23913.yml

```diff
@@ -0,0 +1,72 @@
+title: DOM Based Cross-site Scripting in rails-ujs for contenteditable HTML Elements
+date: 2023-03-13
+description: |
+
+  NOTE: rails-ujs is part of Rails/actionview since 5.1.0.
+
+  There is a potential DOM based cross-site scripting issue in rails-ujs
+  which leverages the Clipboard API to target HTML elements that are
+  assigned the contenteditable attribute. This has the potential to occur
+  when pasting malicious HTML content from the clipboard that includes a
+  data-method, data-remote or data-disable-with attribute.
+
+  This vulnerability has been assigned the CVE identifier CVE-2023-23913.
+
+  Versions Affected: >= 5.1.0 Not affected: < 5.1.0
+  Fixed Versions: 6.1.7.3, 7.0.4.3
+
+  Impact
+    If the specified malicious HTML clipboard content is provided to a
+    contenteditable element, this could result in the arbitrary execution
+    of javascript on the origin in question.
+
+  Releases
+    The FIXED releases are available at the normal locations.
+
+  Workarounds
+    We recommend that all users upgrade to one of the FIXED versions.
+    In the meantime, users can attempt to mitigate this vulnerability
+    by removing the contenteditable attribute from elements in pages
+    that rails-ujs will interact with.
+
+  Patches
+    To aid users who aren’t able to upgrade immediately we have provided
+    patches for the two supported release series. They are in git-am
+    format and consist of a single changeset.
+
+  * rails-ujs-data-method-contenteditable-6-1.patch - Patch for 6.1 series
+  * rails-ujs-data-method-contenteditable-7-0.patch - Patch for 7.0 series
+
+  Please note that only the 7.0.Z and 6.1.Z series are supported at
+  present, and 6.0.Z for severe vulnerabilities.
+
+  Users of earlier unsupported releases are advised to upgrade as soon
+  as possible as we cannot guarantee the continued availability of
+  security fixes for unsupported releases.
+
+  Credits
+    We would like to thank ryotak 15 for reporting this!
+
+  * rails-ujs-data-method-contenteditable-6-1.patch (8.5 KB)
+  * rails-ujs-data-method-contenteditable-7-0.patch (8.5 KB)
+  * rails-ujs-data-method-contenteditable-main.patch (8.9 KB)
+
+cvss_v3: 7.5
+patched_versions:
+ - '~> 6.1.7.3'
+ - '>= 7.0.4.3'
+related:
+ url:
+  - https://discuss.rubyonrails.org/t/cve-2023-23913-dom-based-cross-site-scripting-in-rails-ujs-for-contenteditable-html-elements/82468
+  - https://vulmon.com/vendoradvisory?qidtp=debian_cvelist_bugreportlogs&qid=aa67066c383f12dee5aee964667de4d8
+  - https://github.com/cloudsecurityalliance/gsd-database/blob/main/2023/29xxx/GSD-2023-29013.json
+  - https://access.redhat.com/security/cve/cve-2023-23913
+  - https://bugzilla.redhat.com/show_bug.cgi?id=2182160
+  - https://github.com/cloudsecurityalliance/gsd-database/blob/main/2023/29xxx/GSD-2023-29013.json
+
+# NOTES: No values for [library, platform, osvdb, ghsa, unaffected_versions] fields.
```

Comment:
Also the upstream advisory now says `Not affected: < 5.1.0`. `unaffected_versions` should be added.